arrays/array_practice.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+
+numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+puts numbers.join('...') + '...'
+puts "T-#{numbers[0..9].reverse.join(', ')}...  BLASTOFF!"
```

Comment:
No need for the `[0-9]`.  `puts "T-#{numbers.reverse.join(', ')}...  BLASTOFF!"` works...